<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-26T12:26:22 IST" name="Default suite" finished-at="2024-10-26T12:27:34 IST" duration-ms="71381">
    <groups>
      <group name="Master">
        <method signature="TC_001AcountsRegistrationTest.verify_account_registration()[pri:0, instance:com.testCases.TC_001AcountsRegistrationTest@320de594]" name="verify_account_registration" class="com.testCases.TC_001AcountsRegistrationTest"/>
      </group> <!-- Master -->
      <group name="Regression">
        <method signature="TC_001AcountsRegistrationTest.verify_account_registration()[pri:0, instance:com.testCases.TC_001AcountsRegistrationTest@320de594]" name="verify_account_registration" class="com.testCases.TC_001AcountsRegistrationTest"/>
      </group> <!-- Regression -->
    </groups>
    <test started-at="2024-10-26T12:26:22 IST" name="Default test" finished-at="2024-10-26T12:27:34 IST" duration-ms="71381">
      <class name="com.testCases.TC_001AcountsRegistrationTest">
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:com.testCases.TC_001AcountsRegistrationTest@320de594]" started-at="2024-10-26T12:26:22 IST" name="setUp" finished-at="2024-10-26T12:27:07 IST" duration-ms="44186" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verify_account_registration()[pri:0, instance:com.testCases.TC_001AcountsRegistrationTest@320de594]" started-at="2024-10-26T12:27:07 IST" name="verify_account_registration" finished-at="2024-10-26T12:27:34 IST" duration-ms="27036" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at com.testCases.TC_001AcountsRegistrationTest.verify_account_registration(TC_001AcountsRegistrationTest.java:62)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_account_registration -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.testCases.TC_001AcountsRegistrationTest@320de594]" started-at="2024-10-26T12:27:34 IST" name="tearDown" finished-at="2024-10-26T12:27:34 IST" duration-ms="42" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.testCases.TC_001AcountsRegistrationTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
